rosdep/python.yaml

```diff
@@ -7596,6 +7596,16 @@ python3-pytest-timeout:
   gentoo: [dev-python/pytest-timeout]
   opensuse: [python3-pytest-timeout]
   ubuntu: [python3-pytest-timeout]
+python3-pytrinamic-pip:
+  debian:
+    pip:
+      packages: [pytrinamic]
+  fedora:
+    pip:
+      packages: [pytrinamic]
+  ubuntu:
+    pip:
+      packages: [pytrinamic]
```

Comment:
```suggestion
python3-pytrinamic-pip:
  debian:
    pip:
      packages: [pytrinamic]
  fedora:
    pip:
      packages: [pytrinamic]
  opensuse:
    pip:
      packages: [pytrinamic]
  ubuntu:
    pip:
      packages: [pytrinamic]
```

Reply:
**opensuse:** I added opensuse as requested. @mabelzhang @nuclearsandwich  Thanks for the hint.

**PyTrinamic vs pytrinamic:** @mabelzhang I have had a look at the PyYAML package ([PyPI](https://pypi.org/project/PyYAML/), [pyyaml.org](https://pyyaml.org/)). This package also contains upper case letters in its name and is in the package name with lower case letters. An other example would be `PyVirtualSerialPorts` or `Adafruit-Blinka`, again both habe upper case letters in the name, lower case letters in the package name.
Therefore I would state that lower case letters in the package name are the more consistent choice within the `rosdep.yaml` file.